unstructured/ingest/interfaces.py

```diff
@@ -93,12 +97,35 @@ def write_result(self):
         """Write the structured json result for this doc. result must be json serializable."""
         pass
 
+    def partition_file(self):
+        if not self.config.partition_by_api:
+            logger.debug("Using local partition")
+            elements = partition(filename=str(self.filename))
+            return convert_to_dict(elements)
+
+        else:
+            hostname = self.config.partition_host
+            path = "general/v0/general"
```

Comment:
can this not be hardcoded?
perhaps change partition_host to partition_endpoint?